### PR TITLE
Working Changelogs & CI Improvements

### PR DESCRIPTION
**Pull request by AndrewMontagne**
Changelogs!

image

Also adds the strings, sound and html folders to the build artefact.

:cl: AndrewMontagne
fix: Changelogs should now work!
/:cl:

### CODE REVIEW

**Comment by SamHPurp** [comment]
You are a monster

File: html/templates/header.html [hunk 1]
```diff
@@ -23,35 +23,28 @@
...
 </table>
 
 		<!--
-		
-		TO ADD AN ENTRY, ADD AND MAINTAIN YOUR OWN changelog/USERNAME.yml FILE. 
-		
+
+		TO ADD AN ENTRY, ADD AND MAINTAIN YOUR OWN changelog/USERNAME.yml FILE.
+
 		*** DO NOT FUCK WITH THIS FILE OR YOU WILL CAUSE MERGE CONFLICTS. ***
```
Comment (Allfd) [inline html/templates/header.html]: This is a wise warning.

File: html/templates/header.html [hunk 2]
```diff
@@ -23,35 +23,28 @@
...
-			<font size='2' color='red'><b><br>Have a bug to report?</b> Visit our <a href="https://github.com/tgstation/-tg-station/issues?labels=Bug&state=open">Issue Tracker</a>.<br></font>
-			<font size='2'>Please ensure that the bug has not already been reported and <u><a href="https://github.com/tgstation/-tg-station/issues/724"><b>use the template provided here</b></a></u>.</font>
+			<font size='2'><b>Current Project Maintainers:</b> <a href='https://github.com/OracleStation?tab=members'>-Click Here-</a><br></font>
+			<font size='2'><b>Currently Active GitHub contributor list:</b> <a href='https://github.com/OracleStation/OracleStation/graphs/contributors'>-Click Here-</a><br></font>
+			<font size='2'><b>Main Testers:</b> Anyone who has submitted a bug to the issue tracker<br></font>
+			<font size='2'><b>Thanks to:</b> /tg/station, Paradise Station Baystation 12, /vg/station, NTstation, CDK Station devs, FacepunchStation, GoonStation devs and the original SpaceStation developers.<br> Also a thanks to anybody who has contributed who is not listed here :( Ask to be added here on irc.</font>
```
Comment (FlattestGuitar) [inline html/templates/header.html]: Missed a coma between Paradise Station and Baystation 12.

I'm helping!

**Comment by SamHPurp** [comment]
My requests have been taken care of. But this is with Flat/Alfd to approve - as I don't understand most of it.

**Pull request merged**